src/table/table.scss

```diff
@@ -57,14 +57,14 @@ $hover-scrollbar-color: white !default;
 
   // selection/hover
   &.selectable {
-    .public_fixedDataTableCell_cellContent {
+    .public_fixedDataTableRow_main {
```

Comment:
It's because this rule must override the default background, that is applied to row_odd and row_even